docs/content/development/extensions-contrib/sqlserver.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: doc_page
+---
+
+# Microsoft SQLServer
+
+Make sure to [include](../../operations/including-extensions.html) `sqlserver-metadata-storage` as an extension.
```

Comment:
Mentioned in dev-sync today, can you please link to this doc under https://github.com/druid-io/druid/blob/master/docs/content/development/extensions.md in the contrib section?